### PR TITLE
Fix typo in test-aspnet-core-services-web-apps.md

### PR DESCRIPTION
**Pull request by zakaria-c**
## Summary

Change the indefinite article "a" to the conjunction "and."
Add a comma before the conjunction.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ The reference application (eShopOnContainers) tests were recently restructured a
 
 3. **Application functional/integration tests**, which focus on microservices integration, with test cases that exert several microservices. These tests are located in project **Application.FunctionalTests**.
 
-Unit and integration test per microservice are contained in a test folder in each microservice and Application a Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.
+Unit and integration test per microservice are contained in a test folder in each microservice, and Application and Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.
```
Comment (sughosneo) [inline docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md:138]: Wouldn't that be better to end the sentence after the word "microservice" ? For e.g `....microservice. Application and Load tests ...`

Comment (zakaria-c) [reply docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md:138]: From what I’ve seen across the book, the comma + conjunction is used quite a lot, especially where ideas are linked; plus, I think that opting for a full stop will kind of break the flow of the sentence, though that could be fixed by adding a conjunction after the full stop.
Thanks @sughosneo

Comment (zakaria-c) [reply docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md:138]: What do you think of this @sughosneo ?
While unit and integration tests are contained in a test folder in each microservice, Application and Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.

Comment (sughosneo) [reply docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md:138]: > What do you think of this @sughosneo ?
> While unit and integration tests are contained in a test folder in each microservice, Application and Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.

@zakaria-c , this looks good. One more final comment, we should change `Application` to "application" and `Load` to "load" ..

Comment (zakaria-c) [reply docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md]: @sughosneo , ok I've just updated it; thx!

Comment (sughosneo) [reply docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md]: Looks good. Thanks @zakaria-c . We will merge the changes.

**Comment by sughosneo** [comment]
Hey @zakaria-c , while reviewing this PR with @nishanil when we read the sentence again we realized that there is scope of further improvement. So we rephrased it likes this - *"While unit and integration tests are organized in a test folder within the microservice project, application and load tests are managed separately under the root folder, as shown in Figure 6-25"*. Thoughts ? 

Thank you.

**Comment by zakaria-c** [comment]
Clearer and simpler! I'll update it right away.

**Review by nishanil — APPROVED** [review]
Looks great! Thanks @zakaria-c

**Comment by zakaria-c** [comment]
You're very welcome

**Pull request merged**